src/ralph/models/xapi/video/statements.py

```diff
@@ -1,25 +1,150 @@
 """Video xAPI event definitions"""
 
 from ..base import BaseXapiModel
-from .fields.contexts import VideoPlayedContextField
+from .fields.contexts import (
+    VideoCompletedContextField,
+    VideoInitializedContextField,
+    VideoInteractedContextField,
+    VideoPausedContextField,
+    VideoPlayedContextField,
+    VideoSeekedContextField,
+    VideoTerminatedContextField,
+)
 from .fields.objects import VideoObjectField
-from .fields.results import VideoPlayedResultField
-from .fields.verbs import VideoPlayedVerbField
+from .fields.results import (
+    VideoCompletedResultField,
+    VideoInteractedResultField,
+    VideoPausedResultField,
+    VideoPlayedResultField,
+    VideoSeekedResultField,
+    VideoTerminatedResultField,
+)
+from .fields.verbs import (
+    VideoCompletedVerbField,
+    VideoInitializedVerbField,
+    VideoInteractedVerbField,
+    VideoPausedVerbField,
+    VideoPlayedVerbField,
+    VideoSeekedVerbField,
+    VideoTerminatedVerbField,
+)
 
 
-class VideoPlayed(BaseXapiModel):
+class BaseVideoStatement(BaseXapiModel):
+    """Base model for xAPI video statements.
+
+    Attributes:
+        object (dict): See VideoObjectField.
+    """
+
+    object: VideoObjectField
+
+
+class VideoInitialized(BaseVideoStatement):
+    """Represents a video initialized xAPI statement.
```

Comment:
Can you explain a bit more what every event correspond to? E.g. _John launched the video player_